Marlin/ultralcd.cpp

```diff
@@ -958,11 +958,13 @@ void kill_screen(const char* lcd_msg) {
     }
 
     void _lcd_mesh_edit() {
+      lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
       _lcd_mesh_fine_tune(PSTR("Mesh Editor"));
     }
 
     float lcd_mesh_edit() {
       lcd_goto_screen(_lcd_mesh_edit_NOP);
+      lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
```

Comment:
Well...   I don't know if it was a 'Problem'.  But what was happening was the LCD Panel would go totally blank until the encoder wheel was moved one way or the other.    It didn't always blank the display.   But some times it did.   So I put the line into the lcd_mesh_edit() function and the screen always looked correct.